content/downloading/contents.lr

```diff
@@ -24,12 +24,12 @@ If you're unable to download Tor Browser from the official Tor Project website,
 
 GetTor is a service that automatically responds to messages with links to the latest version of Tor Browser, hosted at a variety of locations, such as Dropbox, Google Drive and GitHub.
 
+For more information on the gettor service, see https://gettor.torproject.org/. 
+
 ###### TO USE GETTOR VIA EMAIL:
 
 Send an email to gettor@torproject.org, and in the body of the message simply write “windows”, “osx”, or “linux”, (without quotation marks) depending on your operating system.
 
 GetTor will respond with an email containing links from which you can download the Tor Browser package, the cryptographic signature (needed for verifying the download), the fingerprint of the key used to make the signature, and the package’s checksum. You may be offered a choice of “32-bit” or “64-bit” software: this depends on the model of the computer you are using.
 
-###### TO USE GETTOR VIA JABBER/XMPP (JITSI, COYIM, ETC.):
-
-To get links for downloading Tor Browser in Chinese for Linux, send a message to gettor@torproject.org with the words "linux zh" in it.
+Optionally, you can also specify your locale. For example, To get links for downloading Tor Browser in Chinese for Linux, send a message to gettor@torproject.org with the words "linux zh" in it.
```

Comment:
The email link is formed automatically by lektor. To prevent this you can enclose the XMPP address in ` characters.

Reply:
FYI, XMPP support is now deprecated and this was removed from the Docs.